pkg/ruler/storage.go

```diff
@@ -23,10 +24,11 @@ type RuleStoreConfig struct {
 	ConfigDB client.Config `yaml:"configdb"`
 
 	// Object Storage Configs
-	Azure azure.BlobStorageConfig `yaml:"azure"`
-	GCS   gcp.GCSConfig           `yaml:"gcs"`
-	S3    aws.S3Config            `yaml:"s3"`
-	Swift openstack.SwiftConfig   `yaml:"swift"`
+	S3         s3.Config         `yaml:"s3"`
```

Comment:
Super silly thing. Would you mind keeping the same order as before? Will help to simplify the `configuration-file-reference.md` diff, to easily spot all the changes in the config.

Reply:
Yea, sure thing. I got a bit over eager and decided to make it alphabetical for no reason in particular